Functions/Import/xASL_imp_DCM2NII_ReorderTimeEncoded.m

```diff
@@ -22,28 +25,62 @@ function xASL_imp_DCM2NII_ReorderTimeEncoded(nii_files, bTimeEncoded, resultJSON
 
     if numel(nii_files)>=1
         [resultPath, resultFile] = xASL_fileparts(nii_files{1});
-        % Check if we the current sequence is a Hadamard or not
-        if bTimeEncoded
+        % Check if the current sequence is a Hadamard from FME or not
+		if bTimeEncoded && bTimeEncodedFME
```

Comment:
OK then we just wait until we will have `bTimeEncodedGE` `bTimeEncodedPhilips` and `bTimeEncodedDanny` as well ;) Just saying that you want to separate generalizable and specific functions/code sections, that's all.

This function has a general time encoded name, but works only for FME TimeEncoded, (otherwise it is skipped). So I would then rename this function to *ReorderTimeEncodedFME or something similar.

Reply:
Done